xml/obs_scm_ci_workflow_integration.xml

```diff
@@ -0,0 +1,1044 @@
+            Replace 12345 with the OBS token numerical ID previously
+            obtained.</para>
+          </listitem>
+
+          <listitem>
+            <para><emphasis role="bold">Secret Token:</emphasis>
+            <emphasis>uvwxyz</emphasis> Replace uvwxyz with the OBS token
+            secret string previously obtained.</para>
+          </listitem>
+
+          <listitem>
+            <para><emphasis role="bold">Trigger:</emphasis> <emphasis>Merge
+            request events</emphasis>, <emphasis>Push
+            events</emphasis>.</para>
+          </listitem>
+        </itemizedlist>
+      </sect3>
+
+      <sect3 xml:id="sec.obs.obs_scm_ci_workflow_integration.setup.webhooks.scm_events">
+        <title>SCM Events</title>
+
+        <para>On a GitHub's repository, events are happening all the time: a
+        pull request is created, somebody pushes a commit, a pull request is
+        merged, etc. When you set up a webhook on the SCM side, you can
+        specify which events you are interested in. Only when those events
+        happen, the webhook will be sent to OBS.</para>
+
+        <para>This is the list of SCM events supported by the existing
+        workflows in OBS:</para>
+
+        <itemizedlist>
+          <listitem>
+            <para>Pull requests</para>
+          </listitem>
+
+          <listitem>
+            <para>Pushes</para>
+          </listitem>
+        </itemizedlist>
+
+        <para>Other SCMs like GitLab have similar behaviour; the main
+        difference is how they name the events. Refer to the <link
+        linkend="sec.obs.obs_scm_ci_workflow_integration.setup.equivalence_table">Equivalence
+        Table</link> for more details or read more about <link
+        xlink:href="https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads">GitHub
+        events</link> and <link
+        xlink:href="https://docs.gitlab.com/ce/user/project/integrations/webhooks.html#events">GitLab
+        events</link>.</para>
+      </sect3>
+    </sect2>
+
+    <sect2 xml:id="sec.obs.obs_scm_ci_workflow_integration.setup.obs_workflows">
+      <title>OBS Workflows</title>
+
+      <para>An SCM event occurs and OBS receives the corresponding webhook.
+      Now is when the <emphasis role="bold">OBS workflows</emphasis> come into
+      play.</para>
+
+      <para>A workflow is nothing else than a sequence of <emphasis
+      role="bold">steps</emphasis> or actions the user wants to perform in
+      OBS. The user can describe the steps to run in a YAML configuration
+      file.</para>
+
+      <para>To do so, in the root directory of your GitHub repository, create
+      a directory <emphasis>.obs</emphasis> which contains a file called
+      <emphasis>workflows.yml</emphasis>.</para>
+
+      <para>The content of <emphasis>.obs/workflows.yml</emphasis> could look
+      like this:<screen>main_workflow:
```

Comment:
I think you should start out with an easier example. One workflow, one step. Then explain multiple workflows later...